src/library.js

```diff
@@ -752,10 +752,6 @@ LibraryManager.library = {
     abort('Assertion failed: ' + UTF8ToString(condition) + ', at: ' + [filename ? UTF8ToString(filename) : 'unknown filename', line, func ? UTF8ToString(func) : 'unknown function']);
   },
 
-  __assert_func: function(filename, line, func, condition) {
```

Comment:
I thought I remembered this was used once, but looks like it's `__assert_fail` and not `func`...

Reply:
Yup